pyvista/plotting/renderer.py

```diff
@@ -1047,6 +1047,7 @@ def show_bounds(
         self,
         mesh=None,
         bounds=None,
+        axes_ranges=None,
```

Comment:
I noticed that `axes_ranges` is no longer an attribute on the `Plotter`. I assume this is on purpose, and I assume because `renderer.bounds` probably also doesn't reflect custom `bounds` being passed to `show_bounds()`. But it's better to discuss this change explicitly.

Reply:
That will have to be a plotter property if we add it, so we can document it. Do you think we need this?

Reply:
I remembered discussing this earlier, in https://github.com/pyvista/pyvista/pull/2441#discussion_r847667283. But apparently @JacobBumgarner ended up thinking that it's not worth having in the `Plotter` API, and I'd be on the fence myself, so I think this can stay as is if you agree.

Reply:
Yes that's correct.  I decided that adding the `axes_ranges` attribute to the plotter would only bloat it, as I image the use case is quite small. 

Moreover, the property is only relevant when a vtkCubeAxesActor was added to the plotter.